### DISCUSSION TITLE
[Feature] Invert Selection in de-duplicator

### DISCUSSION THREAD

**Discussion by Skivling** [immich-app/immich#14717] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

There should be a button and keyboard shortcut to invert the selection of what is trash and keep: keep becomes trash, trash becomes keep. It would sit next to 'Select keep all' and 'Select trash all' buttons. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by RobbertTech** [comment]
I was just thinking about this while I was cleaning. I have a lot of duplicate JPG and HEIC files, and in most cases the JPG is selected by default to keep.
